.github/workflows/sonarcloud.yml

```
@@ -0,0 +1,94 @@
+              uses: actions/setup-dotnet@v3
+              with:
+                  dotnet-version: "8.0.x"
+
+            - name: Set up JDK 17 for SonarCloud
+              uses: actions/setup-java@v3
+              with:
+                  java-version: 17
+                  distribution: "zulu"
+
+            - name: Cache SonarCloud packages
+              uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)